client/src/modules/plan/planner/idea/idea.js

```diff
@@ -0,0 +1,38 @@
+import angular from 'angular';
+
+// import child dependencies
+
+// imports for this component
+import template from './idea.html';
+import './idea.css';
+
+class IdeaController {
+  constructor(EventService, $stateParams) {
+    this.event = {
```

Comment:
pull test data into constants declared outside the controller, that way it's easy to see them as separate from

i.e.: const TEST_DATA = { jon's event}..

this.title = TEST_DATA.title;
this.start.....